src/electron/electron/core.cljs

```diff
@@ -118,6 +118,17 @@
                            (fn [part]
                              (. fs copy (path/join app-path part) (path/join static-dir part)))
                            ["css" "fonts" "icons" "img" "js"])))
+                ;;
+                ;; The "css" and "js" directories are copied from the app bundle,
+                ;; which could be on a read-only file system. Ensure they can be
+                ;; written to so that the rest of the export succeeds.
+                ;;
+                ;;   https://github.com/logseq/logseq/issues/5848
+                ;;   https://github.com/logseq/logseq/issues/6880
+                ;;
```

Comment:
nit: The commit will link to the PR, maybe just mention these issues in the PR instead.